packages/es-components/src/components/containers/fieldset/Fieldset.js

```diff
@@ -1,37 +1,54 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
-const StyledFieldset = styled.fieldset`
-  border: 0;
+import OrientationContext from '../../controls/OrientationContext';
+import genName from '../../util/generateAlphaName';
+
+const FieldsetBase = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
   margin-bottom: 25px;
-  padding: 0;
+  width: 100%;
 
   > *:last-child {
     margin-bottom: 0;
   }
+
+  ${props =>
+    props.orientation === 'inline' &&
+    css`
+      @media (min-width: ${props.theme.screenSize.tablet}) {
+        align-items: baseline;
+        flex-direction: row;
+      }
+    `};
 `;
 
-const Legend = styled.legend`
-  border: 0;
-  border-bottom: 1px solid ${props => props.theme.colors.gray6};
+const Legend = styled.div`
```

Comment:
Is changing this from a `legend` to a `div` going to have any accessibility considerations we have to take into account?

Reply:
because the `Legend` below has an id and the `Fieldset` uses `aria-labelledby`, this should address any potential concerns.